templates/index.html

```diff
@@ -2,17 +2,31 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
-    </div>
-  </div>
+<div class="col-md-12">
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th scope="col">id</th>
+        <th scope="col">Title</th>
+        <th scope="col">Status</th>
+        <th scope="col">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for item in items %}
+      <tr>
+        <th scope="row">{{ item['id']}}</th>
+        <td>{{ item['title']}}</td>
+        <td>{{ item['status'] }}</td>
+        <td>
+          <form class="form-inline" action="" method="PUT">
+            <button type="submit" class="btn btn-success" name="CompleteTask"><i class="fa fa-check" aria-hidden="true"></i>&nbspComplete</button>
+            <button type="submit" class="btn btn-danger" name="CompleteTask"><i class="fa fa-trash" aria-hidden="true" disabled></i>&nbspDelete</button>
+          </form>
```

Comment:
- An HTML form only supports method="POST" or "GET".
- The Delete and Complete buttons should be separate forms.
- It's nicer to use styling to create the spacing rather than &nbsp